int16[] position
---
float32 depth